src/ig/IGExporter.ts

```diff
@@ -131,57 +115,68 @@ export class IGExporter {
         parameter: [
           {
             code: 'copyrightyear',
-            value: '' // Gets set when ig.ini is processed
+            value:
+              this.config?.parameters?.find(p => p.code == 'copyrightyear')?.value ??
+              `${new Date().getFullYear()}+`
           },
           {
             code: 'releaselabel',
-            value: '' // Gets set when ig.ini is processed
+            value: this.config?.parameters?.find(p => p.code == 'releaselabel')?.value ?? 'CI Build'
           },
-          {
-            code: 'show-inherited-invariants',
-            value: 'false' // TODO: Make this configurable
-          }
+          ...(this.config?.parameters?.filter(
+            p => ['copyrightyear', 'releaselabel'].indexOf(p.code) == -1
+          ) ?? [])
```

Comment:
Does it crash?  If so, then I vote we make it required in the config.  I just don't want to guess at something we don't actually know.

Reply:
![image](https://user-images.githubusercontent.com/52747882/80243672-a4883500-8635-11ea-9b53-e8a15a8f0f48.png)

I got the following when removing the `copyrightyear` parameter. This is just using `fhir.base.template`.

Reply:
And similar for `releaselabel`:
![image](https://user-images.githubusercontent.com/52747882/80243846-f204a200-8635-11ea-90b4-afdf77cefd4f.png)

Reply:
Although I do not get a crash when removing `show-inherited-invariants`, even though this comment here kinda makes me think I should:
![image](https://user-images.githubusercontent.com/52747882/80244790-93402800-8637-11ea-9ac4-4d3f8b469600.png)

Reply:
So I think we should make `copyrightyear` and `releaselabel` required then.  I'll do that in my branch.  I think that comment (above `parameter: [`) was written when `parameter` only had the `copyrightyear` and `releaselabel` items.  I added `show-inherited-invariants` later and so that comment doesn't really apply.  That was more of a "user friendly default" at the time.